src/main/java/com/cglx/Calculator.java

```diff
@@ -37,5 +37,23 @@ public int square(int a) {
 		return result;
 	}
 
+	public int sqroot(int a) {
+
+		return (int) Math.sqrt(a);
+	}
+
+	private void readFile() throws IOException {
+		BufferedReader reader = new BufferedReader(new FileReader("testfile.txt"));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "BufferedReader" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AS2095)